jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Build.java

```diff
@@ -38,24 +43,73 @@
 
   @CommandLine.Spec private CommandSpec spec = CommandSpec.create();
 
+  @CommandLine.Mixin
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  CommonCliOptions commonCliOptions;
+
   @CommandLine.ParentCommand
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   protected JibCli globalOptions;
```

Comment:
Maybe we can remove this field? `JibCli` doesn't have any options anymore, right?

Reply:
Good catch.